src/components/graph/menus/dynamic-simulation/event-modification-scenario-editor.tsx

```diff
@@ -50,23 +49,22 @@ import {
 } from '../network-modification-node-editor';
 import { EQUIPMENT_TYPE_LABEL_KEYS } from '../../util/model-constants';
 import { areUuidsEqual } from 'components/utils/utils';
+import { AppDispatch } from '../../../../redux/store';
 
 const EventModificationScenarioEditor = () => {
     const intl = useIntl();
     const notificationIdList = useSelector(
-        (state: ReduxState) => state.notificationIdList
+        (state: AppState) => state.notificationIdList
     );
     const params = useParams();
     const studyUuid = params?.studyUuid
         ? decodeURIComponent(params.studyUuid)
         : undefined;
     const { snackError } = useSnackMessage();
     const [events, setEvents] = useState<Event[]>([]);
-    const currentNode = useSelector(
-        (state: ReduxState) => state.currentTreeNode
-    );
+    const currentNode = useSelector((state: AppState) => state.currentTreeNode);
 
-    const currentNodeIdRef = useRef<UUID>(); // initial empty to get first update
+    const currentNodeIdRef = useRef<string>(); // initial empty to get first update
```

Comment:
I override the `id: string;` of `Node<>` to match the `UUID` type we use for our nodes. Else each time we retrieve data from redux tsc throw an error because `string` cannot be passed to `UUID` variable (functions and components).